internal/core/regularsurface.hpp

```diff
@@ -62,7 +62,7 @@ class RegularSurface{
     {}
 
     /* Grid position (row, col) -> world coordinates */
-    Point coordinate(
+    Point toCDP(
```

Comment:
`toCDP` is follows the golang naming scheme. A more c++-ish name would be `to_cdp`